src/main/java/com/ldtteam/structurize/placement/handlers/placement/PlacementHandlers.java

```diff
@@ -98,6 +99,29 @@ public static void add(IPlacementHandler handler)
         handlers.add(1, handler);
     }
 
+    /**
+     * Finds the appropriate {@link IPlacementHandler} for the given location.
+     * @param world     the world.
+     * @param worldPos  the world position.
+     * @param newState  the blockstate being placed or removed.
+     * @return          the appropriate handler, or null.
+     */
+    @Nullable
+    public static IPlacementHandler getHandler(@NotNull final Level world,
+                                               @NotNull final BlockPos worldPos,
+                                               @NotNull final BlockState newState)
```

Comment:
I assume notnull by default, it's kinda noisy to have it everywhere

Reply:
It's mostly our code style tbh